cmd/model/write_test.go

```diff
@@ -22,7 +23,14 @@ func TestWriteInvalidModel(t *testing.T) {
 	mockFgaClient := mockclient.NewMockSdkClient(mockCtrl)
 	modelString := "{bad_json"
 
-	_, err := write(mockFgaClient, modelString)
+	model := authorizationmodel.AuthzModel{}
+
+	err := model.ReadFromJSONString(modelString)
+	if err != nil {
+		return
```

Comment:
Why do we return?  If we expect error, should we assert on it?

Reply:
Moved this test to the model file